src/main/java/Lotto.java

```diff
@@ -0,0 +1,28 @@
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Lotto {
+    private static final List<LottoNumber> lottoNumbers = IntStream.rangeClosed(1, 45)
+            .boxed()
+            .map(LottoNumber::of)
+            .collect(Collectors.toList());
+
+    private final LottoNumbers value;
+
+    public Lotto() {
+        this.value = LottoNumbers.fromLottoNumberList(lottoNumbers);
+    }
+
+    public Lotto(LottoNumbers value) {
```

Comment:
```suggestion
    public Lotto(List<Integer> value) {

    public Lotto(List<LottoNumber> value) {
```
생성자에 위와 같은 타입을 가지는 생성자도 추가하면 어떨까?

Reply:
제네릭 타입소거로 인해 위와 같이 List를 인자로 갖는 생성자를 2개 이상 생성할 수가 없습니다. 
해결책은 새로운 도메인 클래스 분리일까요?